Chapter3/src/wci/frontend/java/JavaScanner.java

```diff
@@ -76,7 +76,14 @@ private void skipWhiteSpace() throws Exception {
 					} while ((currentChar != EOL) && (currentChar != EOF));
 				}
 				else if (nextChar == '*') { //multi-line comment
-					//todo
+					do {
+						currentChar = nextChar();
+					} while ((currentChar != '/') && (currentChar != EOF));
+					
+					// reach end of multiline comment
+					if (currentChar == '/') {
```

Comment:
Does this mean /* this is a multiline comment/ ? We should be checking for */ as ending. 
See https://github.com/alexliu16/cs-153/pull/5/files how I went about it. I'm not a fan of my source.peekChar() calls.